.travis.yml

```diff
@@ -41,8 +41,12 @@ jobs:
 
     - stage: docs
       python: "3.7"
+      before_install:
+        - sudo apt-get install pandoc
       install:
         - pip install --upgrade sphinx
+        - pip install --upgrade nbsphinx
+        - pip install --upgrade IPython
```

Comment:
Could you make a `requirements-docs.txt` instead of cramming it all here? That way we can do `pip install -U -r requirements-docs.txt`. That way it's also clear *why* those packages need to be installed, and it helps third parties to find all dependencies.